tests/test_rdm.py

```diff
@@ -501,166 +444,6 @@ def test_calc_rdm_movie_binned(self):
         assert rdm.rdm_descriptors['time'][0] == np.mean(time[:3])
 
 
-class TestCompareRDM(unittest.TestCase):
```

Comment:
@HeikoSchuett I think something went wrong here during a merge; large parts of test_rdm are not on this branch anymore; the commit is probably 75633df0741392b3f0445b172619260cc34808c7

Would it make more sense to try to fix this branch or start over from a clean branch? You'd remember best if a lot of effort went into this already.